notebooks/advanced_RAG.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>What are other splitting techniques? What are their pros and cons? There are some nice suggestions here -<a href="https://www.pinecone.io/learn/chunking-strategies/" target="_blank">https://www.pinecone.io/learn/chunking-strategies/</a> Maybe we could include some of those considerations here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Added considerations on this, and a link to Greg Kamradt's notebook.